rasa/exceptions.py

```diff
@@ -67,3 +67,10 @@ def __str__(self) -> Text:
 
 class ActionLimitReached(RasaException):
     """Raised when predicted action limit is reached."""
+
+
+class ClassificationReportException(RasaException):
```

Comment:
`RasaException` means it doesn't get raised to sentry - as this is something which is _not_ supposed to happen it in my opinion should not subclass from `RasaException` as we want it to appear in Sentry, what do you think?